dev-util/cmake/cmake-3.7.2.recipe

```diff
@@ -0,0 +1,77 @@
+SUMMARY="A cross-platform build system"
+DESCRIPTION="CMake is a family of tools designed to build, test and package \
+software. It is used to control the software compilation process using \
+simple platform and compiler independent configuration files. CMake generates \
+native makefiles and workspaces that can be used in the compiler environment \
+of your choice."
+HOMEPAGE="https://cmake.org/"
+COPYRIGHT="2002-2016 Kitware, Inc., Insight Consortium"
+LICENSE="CMake"
+REVISION="1"
+SOURCE_URI="https://cmake.org/files/v3.7/cmake-$portVersion.tar.gz"
+CHECKSUM_SHA256="dc1246c4e6d168ea4d6e042cfba577c1acd65feea27e56f5ff37df920c30cae0"
+SOURCE_DIR="cmake-$portVersion"
+PATCHES="cmake-$portVersion.patchset"
+
+ARCHITECTURES="!x86_gcc2 x86 x86_64 arm"
+SECONDARY_ARCHITECTURES="x86"
+
+PROVIDES="
+	cmake$secondaryArchSuffix = $portVersion compat >= 3
+	cmd:cmake = $portVersion compat >= 3
+	cmd:ccmake = $portVersion compat >= 3
+	cmd:cpack = $portVersion compat >= 3
+	cmd:ctest = $portVersion compat >= 3
+	"
+REQUIRES="
+	haiku$secondaryArchSuffix
+	lib:libcurl$secondaryArchSuffix
+	lib:libncurses$secondaryArchSuffix
+	lib:libz$secondaryArchSuffix
+	"
+
+BUILD_REQUIRES="
+	haiku${secondaryArchSuffix}_devel
+	devel:libcurl$secondaryArchSuffix
+	devel:libncurses$secondaryArchSuffix
+	devel:libz$secondaryArchSuffix
+	"
+BUILD_PREREQUIRES="
+	cmd:gcc$secondaryArchSuffix
+	cmd:gfortran$secondaryArchSuffix
+	cmd:grep
+	cmd:ld$secondaryArchSuffix
+	cmd:libtoolize$secondaryArchSuffix
+	cmd:make
+	cmd:sed
+	"
+
+BUILD()
+{
+	# not an autotools configure
+	./configure --prefix=$prefix \
+		--datadir=/$relativeDataDir/cmake \
+		--docdir=/$relativeDocDir \
+		--mandir=/$relativeManDir \
```

Comment:
As is noted in the recipe, this is "not an autotools configure" and the arguments are slightly different. IIRC the configure script will convert them to absolute paths and then call the bootstrap cmake with them.